tests/data/EventFactory.ts

```diff
@@ -24,21 +24,45 @@ export class EventFactory {
 
   public static fake(): EventModel {
     const [start, end] = EventFactory.randomTime();
+    return EventFactory.fakeWithTime(start, end);
+  }
+
+  private static fakeWithTime(start: Date, end: Date): EventModel {
+    const event = EventFactory.fakeWithoutTime();
+    event.start = start;
+    event.end = end;
+    return event;
+  }
+
+  public static fakeWithoutTime(): EventModel {
```

Comment:
What's the different between this and `fake`? Or does this method exist because of `fakeWithTime`?

Reply:
As mentioned above, this method is not needed anymore

Reply:
But the idea before was to have `fake, fakePastEvent, fakeOngoingEvent, fakeFutureEvent` to all call `fakeWithTime`, but specifying different times for each section. `fakeWithTime` would then call `fakeWithoutTime` to get a random object for all other fields